test/array.findIndex.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const metasync = require('..');
+const metatests = require('metatests');
+
+metatests.test('findIndex with error', test => {
```

Comment:
Tests are good 👍 
Just nitpicking a grammar, not that important ;-)
```suggestion
metatests.test('findIndex with an error', test => {
```